Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -128,12 +128,14 @@ + (NSString *)testEnvironmentWithConfiguration:(BPConfiguration *)config {
         BPXCTestFile *xctTestFile = [BPXCTestFile BPXCTestFileFromExecutable:executable isUITestFile:(config.testRunnerAppPath == nil) withError:&error];
         NSAssert(xctTestFile != nil, @"Failed to load testcases from %@", [error localizedDescription]);
         NSMutableSet *testsToSkip = [[NSMutableSet alloc] initWithArray:xctTestFile.allTestCases];
-        NSSet *testsToRun = [[NSSet alloc] initWithArray:config.testCasesToRun];
+        NSSet *testsToRun = [[NSSet alloc] initWithArray:[xctTestFile expandTests:config.testCasesToRun]];
+        NSLog(@"testsToRun: %@", testsToRun);
```

Comment:
I actually removed a NSLog here, but it still shows up even though when I edit the file on Github, it does not show up.